lib/howl/commands/file_commands.moon

```diff
@@ -39,6 +39,31 @@ command.register
   input: -> interact.select_file allow_new: true
   handler: (file) -> app\open_file file
 
+command.register
```

Comment:
@nilnor - this branch was merged but the latest change to the command didn't get in? Specifically the buffers are now displayed and the title is not updated.